src/UserSpaceInstrumentation/AllocateInTracee.h

```diff
@@ -14,9 +14,11 @@
 namespace orbit_user_space_instrumentation {
 
 // Allocate `size` bytes of memory in the tracee's address space using mmap. The memory will have
-// read, write, and execute permissions.
+// read, write, and execute permissions. The memory allocated will start at `address`. If `address`
+// is null the placement of memory will be arbitray (compare the documenation of mmap:
```

Comment:
As `address` is a `uint64_t`, technically "if address is zero".

Reply:
done